util/kvdb-rocksdb/src/lib.rs

```diff
@@ -265,31 +266,35 @@ impl Database {
 	/// Open database file. Creates if it does not exist.
 	pub fn open(config: &DatabaseConfig, path: &str) -> Result<Database> {
 		let mut opts = Options::new();
+
 		if let Some(rate_limit) = config.compaction.write_rate_limit {
 			opts.set_parsed_options(&format!("rate_limiter_bytes_per_sec={}", rate_limit))?;
 		}
-		opts.set_parsed_options(&format!("max_total_wal_size={}", 64 * 1024 * 1024))?;
-		opts.set_parsed_options("verify_checksums_in_compaction=0")?;
-		opts.set_parsed_options("keep_log_file_num=1")?;
-		opts.set_max_open_files(config.max_open_files);
-		opts.create_if_missing(true);
 		opts.set_use_fsync(false);
-		opts.set_db_write_buffer_size(DB_WRITE_BUFFER_SIZE);
+		opts.create_if_missing(true);
+		opts.set_max_open_files(config.max_open_files);
+		opts.set_parsed_options("keep_log_file_num=1")?;
+		opts.set_parsed_options("bytes_per_sync=1048576")?;
+		opts.set_db_write_buffer_size(config.memory_budget_per_col() / 2);
+		opts.increase_parallelism(cmp::max(1, ::num_cpus::get() as i32 / 2));
```

Comment:
I'd recommend setting it to `num_cpus - 1`  if there is more than one CPU. One CPU should be always available for block import job to minimize import latency.

Reply:
We set it to `num_cpus / 2` currently, so only using half of the cores (which makes sense to me)